src/Nethermind/Nethermind.Runner/Properties/launchSettings.json

```diff
@@ -86,6 +86,10 @@
     "Baikal": {
       "commandName": "Project",
       "commandLineArgs": "--config configs\\baikal.cfg --Init.DiagnosticMode MemDb"
+    },
```

Comment:
Do we want to remove Aulet from launchSettings?

Reply:
Done, thx